mgrctl/cmd/proxy/config.go

```diff
@@ -0,0 +1,289 @@
+
+  Create a proxy configuration file with generated certificates providing all parameters
+	
+    $ mgrctl proxy create config --proxyName="proxy.example.com" --server="server.example.com" --email="admin@org.com" --caCrt="ca.pem" --caKey="caKey.pem" --caPassword="pass.txt" --cnames="proxy_a.example.com,proxy_b.example.com,proxy_c.example.com" --country="DE" --state="Bayern" --city="Nuernberg" --org="orgExample" --orgUnit="orgUnitExample" --sslEmail="sslEmail@example.com" -o="proxy-config"
+
+  or an alternative format:
+	
+	$ mgrctl proxy create config --proxyName="proxy.example.com" --server="server.example.com" --email="admin@org.com" --caCrt="ca.pem" --caKey="caKey.pem" --caPassword="pass.txt" --cnames="proxy_a.example.com" --cnames="proxy_b.example.com" --cnames="proxy_c.example.com" --country="DE" --state="Bayern" --city="Nuernberg" --org="orgExample" --orgUnit="orgUnitExample" --sslEmail="sslEmail@example.com" -o="proxy-config"`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return utils.CommandHelper(globalFlags, cmd, args, &flags, proxyCreateConfigInit)
+		},
+	}
+
+	addFlags(createConfigCmd)
+
+	// validations
+	utils.MarkMandatoryFlags(createConfigCmd, ProxyCreateConfigRequiredFields[:])
+	createConfigCmd.MarkFlagsOneRequired(proxyCrt, caKey)
+	createConfigCmd.MarkFlagsMutuallyExclusive(proxyCrt, caKey)
+
+	return createConfigCmd
+}
+
+func proxyCreateConfigInit(globalFlags *types.GlobalFlags, flags *ProxyCreateConfigFlags, cmd *cobra.Command, args []string) error {
+	return ProxyCreateConfig(flags, api.Init, proxy.ContainerConfig, proxy.ContainerConfigGenerate)
+}
+
+// ProxyCreateConfig command handler.
+func ProxyCreateConfig(
+	flags *ProxyCreateConfigFlags,
+	apiInit func(*api.ConnectionDetails) (*api.APIClient, error),
+	proxyConfig func(client *api.APIClient, request proxy.ProxyConfigRequest) (*[]int8, error),
+	proxyConfigGenerate func(client *api.APIClient, request proxy.ProxyConfigGenerateRequest) (*[]int8, error),
+) error {
+	client, err := apiInit(&flags.ConnectionDetails)
+	if err == nil {
+		err = client.Login()
+	}
+
+	if err != nil {
+		return utils.Errorf(err, L("failed to connect to the server"))
+	}
+
+	// handle CA certificate path
+	caCertificate := string(utils.ReadFile(flags.CaCrt))
+
+	// Check if ProxyCrt is provided to decide which configuration to run
+	var data *[]int8
+	if flags.ProxyCrt != "" {
+		data, err = handleProxyConfig(client, flags, caCertificate, proxyConfig)
+	} else {
+		data, err = handleProxyConfigGenerate(client, flags, caCertificate, proxyConfigGenerate)
+	}
+
+	if err != nil {
+		return utils.Errorf(err, L("failed to execute proxy configuration api request"))
+	}
+
+	filename := GetFilename(flags.Output, proxyName)
+	if err := utils.SaveBinaryData(filename, *data); err != nil {
+		return utils.Errorf(err, L("error saving binary data: %v"), err)
+	}
+	log.Debug().Msgf("Proxy configuration file saved as %s", filename)
```

Comment:
```suggestion
	log.Info().Msgf("Proxy configuration file saved as %s", filename)
```

Without this when you execute the command it will pause for couple seconds and exit without printing anything.